CONTRIBUTING.md

```diff
@@ -96,343 +61,137 @@ generation.
-  + `fls-shake` for building artifacts
-  + `cabal-install` and `hpack` for Haskell builds
-
-  ```bash
-  nix-shell -A devShells.ci
-  ```
-  or with flakes:
-  ```bash
-  nix develop .#ci
-  ```
-
-+ `devShells.docs`
-
-  Complete documentation publishing environment for mkdocs and mdbook workflows. Includes everything from the default shell plus:
-
-  + `pandoc` for document conversion
-  + full `latex` environment
-  + `python311` with mkdocs, material theme, and extensions
-  + `mdbook` + `cargo` for mdbook ecosystem
-  + `chromium` for PDF rendering
-  + additional build tools
+---
 
-  ```bash
-  nix-shell -A devShells.docs
-  ```
-  or with flakes:
-  ```bash
-  nix develop .#docs
-  ```
+## Development Environment Setup
 
----
+We provide several development shells tailored for different tasks. You can enter them using `nix develop`.
 
-## Usage Instructions
+### Default Shell
 
-### Building Everything
+This is the primary environment for Agda development. It includes Agda, all required libraries, and the `fls-shake` build tool.
 
 ```bash
-# Build all derivations
-nix-build
-
-# Check it completed successfully
-ls -la result*
+# Enter the default development shell
+nix develop
 ```
 
-### Building Specific Targets
+**Available Tools**
 
-```bash
-# Type-check the formal ledger:
-nix-build -A formalLedger
++ `agda` (with all project libraries)
++ `fls-shake` (custom build tool)
++ `python311`
++ `hpack`
 
-# Generate HTML documentation:
-nix-build -A html
+### CI Shell
 
-# Generate Haskell code:
-nix-build -A hsSrc
+A minimal environment designed for automated builds, containing only the `fls-shake` build tool and its runtime dependencies.
 
-# Build PDF documentation:
-nix-build -A docs.conway.fullspec
+```bash
+# Enter the CI shell
+nix develop .#ci
 ```
 
-### Using Nix Flakes
+### Documentation Shell
 
-If you prefer the modern flakes interface:
+A comprehensive environment for working on documentation, including the full documentation generation pipeline.
 
 ```bash
-# Build default package (formalLedger)
-nix build
-
-# Enter default development shell
-nix develop
-
-# Enter specific development shells
-nix develop          # Default development environment
-nix develop .#ci     # Minimal CI environment
-nix develop .#docs   # Complete docs publishing environment
-
-# Build specific packages
-nix build .#formalLedger              # Type-check formal ledger
-nix build .#html                      # Generate HTML documentation
-nix build .#hsSrc                     # Generate Haskell code
-nix build .#docs.conway.fullspec      # Full specification PDF
-nix build .#docs.conway.diffspec      # Conway differential PDF
+# Enter the documentation shell
+nix develop .#docs
 ```
 
-### Development Workflow
+**Available Tools:**
 
-#### For Agda Development
+* Everything from the default shell.
+* `pandoc`, a full `latex` distribution.
+* `mkdocs` and its Python dependencies.
+* `mdbook`, `cargo`, and related tools.
```

Comment:
I did not understand how this list is organized.

Reply:
Good catch.  (fixed)